src/Chapter1.hs

```diff
@@ -635,8 +646,14 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
+firstDigit :: Int -> Int
+firstDigit n 
+          | head n == 0 = n
+          | otherwise = firstDigit (head n)
```

Comment:
Perfect! 🧠 
Although, note, that it won't work correctly on negative numbers.
In the first guard, it would return the negative digit, and in the second guard it will hang and work infinitely 🙂

Reply:
Ah, you're right. I messed this up again 😅